.golangci.yml

```diff
@@ -121,6 +122,6 @@ run:
 # golangci.com configuration
 # https://github.com/golangci/golangci/wiki/Configuration
 service:
-  golangci-lint-version: 1.27.x   # Use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.43.x   # Use the fixed version to not introduce new linters unexpectedly
```

Comment:
Can I ask you to also add an updatecli configuration to bump this version.
It should be pretty straightforward https://github.com/golangci/golangci-lint/releases/tag/v1.43.0

Reply:
Don't merge: I'll add a commit with this change

Reply:
PR upgraded: in fact we won't need an automated version.
I've deleted the `service:` section in `.golangci.yml` as per https://github.com/golangci/golangci/wiki/Configuration#service-configuration : we don't need it.
Thus we don't need to specify the golangcitlint version as well.

The only other location where we specify a fixed version for golangcilint is the githubaction but it's tracked by dependandabot.

So it sounds like we should be ready to roll.